### PR TITLE
Ncnewst5

### PR DESCRIPTION
**Pull request by Joelymodevs**

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -96,3 +97,26 @@ describe("/api", () => {
       });
   });
 });
+describe('/api/articles', () => {
+  it('should have all properties of articles with new comment count column', () => {
+    return request(app).get('/api/articles').expect(200).then((result) => {
```
Comment (pippaamy) [inline __tests__/endpoints.test.js:103]: check the length of the array before forEach

File: __tests__/endpoints.test.js [hunk 2]
```diff
@@ -96,3 +97,26 @@ describe("/api", () => {
...
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.comment_count).toBe("number");
+        expect(typeof article.created_at).toBe("string");
+        expect(typeof article.votes).toBe("number");
+        expect(typeof article.article_img_url).toBe("string");
+      })
+    })
+  });
+  it('no article should have a body property', () => {
+    return request(app).get('/api/articles').expect(200).then((result) =>{
```
Comment (pippaamy) [inline __tests__/endpoints.test.js]: here too

File: __tests__/endpoints.test.js [hunk 3]
```diff
@@ -96,3 +97,26 @@ describe("/api", () => {
...
+      result.body.forEach((article) => {
+        expect(article.body).toBe(undefined)
+      })
```
Comment (pippaamy) [inline __tests__/endpoints.test.js]: lets check that it defaults to created_at and descending order

**Review by pippaamy — CHANGES_REQUESTED** [review]
good job, lets add those tests in! you'll have to add some more functionality in your model for this to pass :)

**Review by robert112233 — APPROVED** [review]
Merge, well done!

**Pull request merged**